bugsnag/django/__init__.py

```diff
@@ -45,14 +46,23 @@ def add_django_request_to_notification(notification):
 
     if getattr(request, "session", None):
         notification.add_tab("session", dict(request.session))
-    notification.add_tab("request", {
+    req = {
         'method': request.method,
         'path': request.path,
         'encoding': request.encoding,
         'GET': dict(request.GET),
         'POST': dict(request.POST),
         'url': request.build_absolute_uri(),
-    })
+    }
+    if 'CONTENT_TYPE' in request.META:
+        if request.META['CONTENT_TYPE'] == 'application/json':
+            req["content_type"] = request.META['CONTENT_TYPE']
+            if req["method"] == "POST":
+                body = request.body.decode('utf-8')
+                uo = json.loads(body.replace("'", '"'))
+                req["POST"] = uo
```

Comment:
I'm not sure what `uo` stands for, but I think we can drop it anyway:

```suggestion
                req["POST"] = json.loads(body)
```

Reply:
Fixed in 4b59d6cf5ee86e11ede693f10421c1f95379d044